### PR TITLE
Update to private ruby-2.3.7/rails-3.2.22.13

### PR DESCRIPTION
**Pull request by dougmartin**

### CODE REVIEW

**Comment by dougmartin** [comment]
@scytacki I've pushed the docker image to the private repo.

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,4 @@
...
-
-# Debian 8 (jessie) is no longer supported
-RUN echo "deb [check-valid-until=no] http://archive.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
-RUN sed -i '/deb http:\/\/\(deb\|httpredir\).debian.org\/debian jessie.* main/d' /etc/apt/sources.list
-RUN apt-get -o Acquire::Check-Valid-Until=false update
-
-#
-# update apt libraries
-# RUN apt-get update
+FROM concordconsortium/docker-rails-base-private:ruby-2.3.7-rails-3.2.22.13
```
Comment (scytacki) [inline Dockerfile]: Can you add back in the `apt-get update` so we get the latest nginx package each time docker-hub rebuilds the image.

Comment (scytacki) [reply Dockerfile]: I took care of this.

**Comment by scytacki** [comment]
I set up a private docker-hub repo, and it seems to build this branch fine.

**Pull request merged**